server/s3_object_metadata.h

```diff
@@ -268,7 +268,9 @@ class S3ObjectMetadata : private S3ObjectMetadataCopyable {
   const struct s3_motr_idx_layout& get_part_index_layout() const {
     return part_index_layout;
   }
-
+  // This is temporary change, will be removed once
+  // delete object changes related delete marker are merged
+  virtual bool get_delete_marker() { return false; }
```

Comment:
can't be a const function

Reply:
This API is implemented as part of DeleteObject changes. I will remove this from here before merging. So we can ignore it for now.

Reply:
ok